tap_iterable/streams.py

```diff
@@ -137,8 +129,10 @@ def sync_data_export(self, state):
                         tf.write(item)
                         count += 1
                         tf.write(b'\n')
+                # Fix for TDL-22208
```

Comment:
could you expand this comment with the issue to make it more readible

Reply:
Done